drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -443,8 +441,63 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var progressTemplate = function(ctx) {
+        // Formats a number to 4 decimals without trailing zeros
+        // (1.00 -> '1'; 1.50 -> '1.5'; 1.333333333 -> '1.3333').
+        var formatNumber = function(n) { return n.toFixed(4).replace(/\.?0+$/, ''); };
+        var is_graded = ctx.graded && ctx.max_score > 0;
```

Comment:
Does `ctx.max_score` include the `weight` of the problem?  It should - to allow course teams to mark problems as not part of the grade.